Mathlib/Order/Fin/Tuple.lean

```diff
@@ -13,8 +13,6 @@ import Mathlib.Order.Interval.Set.Defs
 # Order properties on tuples
 -/
 
-assert_not_exists Monoid
```

Comment:
This is a significant change of contract: this line is here because we went to some trouble not to have this file depend on certain parts of the library.

The right fix for triggering an assertion error like this is usually to rearrange your code so that you continue to conform to the contract.

In this case since it is only your proofs (and not statements) that need the new import, you could try to find proofs which do not need the new import but probably the best thing is to move your lemmas. E.g., create a new file with the definition + API for `finRotate`.

Reply:
Good point - fixed by doing some file splitting.